fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/LoanApplicationTerms.java

```diff
@@ -1702,6 +1588,18 @@ public Money getTotalDisbursedAmount() {
         return disbursedAmount;
     }
 
+    public Money getTotalMultiDisbursedAmount() {
+        Money disbursedAmount = Money.zero(getCurrency());
+        if (isMultiDisburseLoan()) {
+            for (DisbursementData disbursement : getDisbursementDatas()) {
+                disbursedAmount = disbursedAmount.plus(disbursement.getPrincipal());
```

Comment:
@josehernandezfintecheandomx Do we need to consider here the "not yet" disbursement details as well?
Many other places the disbursement details are filtered by whether they got already disbursed or it is just expected yet (actualDisbursementDate = null). What do you think?